src/app/ubs/ubs-user/services/user-messages.service.ts

```diff
@@ -30,8 +30,12 @@ export class UserMessagesService implements OnDestroy {
     return this.http.get<number>(`${this.url}/notifications/quantityUnreadenNotifications`);
   }
 
-  setReadNotification(id: number): Observable<any> {
-    return this.http.post(`${this.url}/notifications/${id}?lang=${this.language}`, {});
+  setReadNotification(id: number): Observable<void> {
```

Comment:
name `setReadNotification` may not accurately reflect its purpose, especially if it specifically marks a notification as read: `updateNotificationStatus`, `markNotificationAsRead` etc